### ISSUE TITLE
create capturable errors

### ISSUE THREAD

**Issue by shyamcody** [geekpradd/PyDictionary#34]
Current version of the code gets error printed. This is problematic while using in other environments. Change the print statements into proper error message generation so that they can be captured using try except or sys and traceback loops. Goal will be:

- [ ] change all error printing into standard error statements

- [ ] add test cases showing that these error statements can be captured and processed using try except or sys traceback loops.

**Comment by Pikamander2** [comment]
I agree. The current method pollutes the console and makes it more difficult to handle the various errors that can arise.

**Comment by rhijjawi** [comment]
See #45
